src/operations/create_collection.ts

```diff
@@ -118,6 +118,10 @@ export class CreateCollectionOperation extends CommandOperation<Collection> {
           const createOp = new CreateCollectionOperation(db, collectionName);
           await createOp.executeWithoutEncryptedFieldsCheck(server, session);
         }
+
+        if (!options.encryptedFields) {
+          this.options = { ...this.options, encryptedFields };
+        }
```

Comment:
Sorry, added this very late but I think this is a definite omission in the spec tests – opened https://github.com/mongodb/specifications/pull/1205 so this doesn’t happen to anybody else

Reply:
And now the spec tests are failing because the `promoteValues` fix from https://github.com/mongodb/libmongocrypt/pull/324 is missing – I think that’s okay for me, unless it’s important that these tests here are skipped until that PR is merged & released

Reply:
I think that's ok